rfcs/0000-snarkworker-socket.md

```diff
@@ -0,0 +1,64 @@
+## Summary
+[summary]: #summary
+
+Pools of Snarkworkers can be made more robust and be load balanced by moving to a persistant socket connection for work requests and responses.
+
+## Terms
+
+snark-coordinator (SC) - daemon(s) participating on the gossip network and queueing up snark-work to be done by snark-workers
+
+snark-worker (SW) - daemon(s) processing compute intensive snarks (snark+libsnark)
```

Comment:
nit: Acronyms

Maybe it's not so bad if they're clearly defined at the top, though?